src/listener.cpp

```diff
@@ -19,7 +19,7 @@
 #include <std_msgs/msg/string.hpp>
 
 
-void chatterCallback(const std_msgs::msg::String::ConstSharedPtr & msg)
+void chatterCallback(const std_msgs::msg::String::ConstSharedPtr msg)
```

Comment:
I think you also need to use `SharedPtr` rather than `ConstSharedPtr` for now. Later we have to figure out how to handle all of these different callback types in a portable way.